### PR TITLE
[xdoctest][task 138 and 140] reformat example code with google style in audio/*

### PR DESCRIPTION
**Pull request by ooooo-create**
### PR types

Others

### PR changes

Others

### Description

修改如下文件的示例代码，使其通过 `xdoctest` 检查：

- `python/paddle/tensor/manipulation.py`
- `python/paddle/tensor/search.py`

`python/paddle/tensor/ops.py` 貌似已经修改过了吗
预览：

- en: http://preview-paddle-pr-XXX.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/jit/XXX_en.html
- zh: http://preview-paddle-pr-XXX.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/jit/XXX_cn.html

### Related links
 
- #55629
- #55295

@sunzhongkai588 @SigureMo @megemini

### CODE REVIEW

File: python/paddle/tensor/manipulation.py [hunk 1]
```diff
@@ -937,10 +940,11 @@ def fill_diagonal_(x, value, offset=0, wrap=False, name=None):
 
     Examples:
         .. code-block:: python
-            import paddle
-            x = paddle.ones((4, 3)) * 2
-            x.fill_diagonal_(1.0)
-            print(x.tolist())   #[[1.0, 2.0, 2.0], [2.0, 1.0, 2.0], [2.0, 2.0, 1.0], [2.0, 2.0, 2.0]]
+            >>> import paddle
```
Comment (SigureMo) [inline python/paddle/tensor/manipulation.py:944]: image

上面需要空一行

**Review by SigureMo — COMMENTED** [review]
其余 LGTM

File: python/paddle/tensor/manipulation.py (hunk 1 above)
Comment (ooooo-create) [reply python/paddle/tensor/manipulation.py:944]: done

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**